packages/transport/src/modules/devices/DevicesController.ts

```diff
@@ -40,15 +40,30 @@ export class DevicesController extends TransportController {
         await this.devices.create(device);
     }
 
-    private async createDevice(name = "", description?: string, isAdmin = false): Promise<Device> {
+    public async sendDevice(parameters: ISendDeviceParameters): Promise<Device> {
+        if (!parameters.name) {
+            const devices = await this.parent.devices.list();
+            parameters.name = `Device ${devices.length + 1}`;
+        }
+
+        const parsedParams = SendDeviceParameters.from(parameters);
+        const device = await this.createDevice(parsedParams);
+
+        await this.devices.create(device);
+
+        return device;
+    }
+
+    private async createDevice(params: SendDeviceParameters): Promise<Device> {
         const [signedChallenge, devicePwdDn] = await Promise.all([this.parent.challenges.createChallenge(ChallengeType.Identity), PasswordGenerator.createStrongPassword(45, 50)]);
 
         this.log.trace("Device Creation Challenge signed. Creating device on backbone...");
 
         const response = (
             await this.client.createDevice({
```

Comment:
Is the backbone notified once the device is onboarded and loses its backup device status?

Reply:
indeed